docs/_basic/authenticating_oauth.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Authenticating with OAuth
```

Comment:
❗  This section needs the most 👀 , by far. Please review this one thoroughly carefully.